docs/zh-cn/user/references/registry/zookeeper.md

```diff
@@ -135,7 +135,8 @@ Zookeeper 集群配置：
 
 ## zookeeper 安装
 
-安装方式参见: [Zookeeper安装手册](../../../admin/install/zookeeper.md)，只需搭一个原生的 Zookeeper 服务器，并将 [Quick Start](../../preface/usage.md) 中 Provider 和 Consumer 里的 `conf/dubbo.properties` 中的 `dubbo.registry.addrss` 的值改为 `zookeeper://127.0.0.1:2181` 即可使用。
+安装方式参见: [Zookeeper安装手册](http://dubbo.apache.org/zh-cn/docs/admin/install/zookeeper.html)，只需搭一个原生的 Zookeeper 服务器，并将 [Quick Start](../../quick-start.md) 中 Provider 和 Consumer 里的 `conf/dubbo.properties` 中的 `dubbo.registry.addrss` 的值改为 `zookeeper://127.0.0.1:2181` 即可使用。
```

Comment:
The link looks good to me, why do you want to change it?

Reply:
Ok，actually two links are the same, choose which one is the same.

Reply:
I'd prefer not change it since it is not broken.

Reply:
ok